src/broker/broker.c

```diff
@@ -172,95 +182,99 @@ static int exit_rc = 0;
 
 #define OPTIONS "+vM:X:k:s:g:EIS:"
 static const struct option longopts[] = {
-    {"verbose",         no_argument,        0, 'v'},
-    {"security",        required_argument,  0, 's'},
-    {"module-path",     required_argument,  0, 'X'},
-    {"k-ary",           required_argument,  0, 'k'},
-    {"heartrate",       required_argument,  0, 'H'},
-    {"shutdown-grace",  required_argument,  0, 'g'},
-    {"setattr",         required_argument,  0, 'S'},
+    {"verbose", no_argument, 0, 'v'},
+    {"security", required_argument, 0, 's'},
+    {"module-path", required_argument, 0, 'X'},
+    {"k-ary", required_argument, 0, 'k'},
+    {"heartrate", required_argument, 0, 'H'},
+    {"shutdown-grace", required_argument, 0, 'g'},
+    {"setattr", required_argument, 0, 'S'},
     {0, 0, 0, 0},
 };
 
 static void usage (void)
 {
     fprintf (stderr,
-"Usage: flux-broker OPTIONS [initial-command ...]\n"
-" -v,--verbose                 Be annoyingly verbose\n"
-" -X,--module-path PATH        Set module search path (colon separated)\n"
-" -s,--security=plain|curve|none    Select security mode (default: curve)\n"
-" -k,--k-ary K                 Wire up in a k-ary tree\n"
-" -H,--heartrate SECS          Set heartrate in seconds (rank 0 only)\n"
-" -g,--shutdown-grace SECS     Set shutdown grace period in seconds\n"
-" -S,--setattr ATTR=VAL        Set broker attribute\n"
-);
+             "Usage: flux-broker OPTIONS [initial-command ...]\n"
+             " -v,--verbose                 Be annoyingly verbose\n"
+             " -X,--module-path PATH        Set module search path (colon separated)\n"
+             " -s,--security=plain|curve|none    Select security mode (default: "
+             "curve)\n"
+             " -k,--k-ary K                 Wire up in a k-ary tree\n"
+             " -H,--heartrate SECS          Set heartrate in seconds (rank 0 only)\n"
+             " -g,--shutdown-grace SECS     Set shutdown grace period in seconds\n"
+             " -S,--setattr ATTR=VAL        Set broker attribute\n");
```

Comment:
Hm, maybe this is case where a `// clang-format off` is desired as part of the PR. This is a special case where we want to see the code as close as possible to the resulting output.

Reply:
I'm not sure I know what you mean.  Is the issue the indentation?